tox.ini

```diff
@@ -5,11 +5,7 @@ envlist = py26,py27,pypy,py33,py34,py35,docs,pep8,py3pep8
 # If you add a new dep here you probably need to add it in setup.py as well
```

Comment:
This comment (and the one in setup.py referencing the tox.ini) are outdated now.